events/management/commands/update_n_events.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management import BaseCommand, CommandError
+from django.db import transaction
+
+from events.models import Keyword, Place, recache_n_events_in_locations, recache_n_events
+
+
+class Command(BaseCommand):
+    help = "Update keyword and place event numbers"
+    models = {'keyword': Keyword,
+              'place': Place}
+
+    def add_arguments(self, parser):
+        parser.add_argument('model', nargs='?', default=False)
+        parser.add_argument('--all',
+                            default=False,
+                            action='store_true',
+                            dest='all',
+                            help='Recalculate everything from scratch')
+
+    def handle(self, *args, **kwargs):
```

Comment:
Just unload `model` and `all` here in the named args. No need to bother with `kwargs.get()`.